hw3/yanagiragi/test.py

```diff
@@ -0,0 +1,77 @@
+import _matrix
+import numpy as np
+import os
+import time
+import pytest, unittest
+
+def measure_time(m, k, n):    
+    testIter = 10
+    elapsed_naive = 0
+    elapsed_mkl = 0
+
+    for idx in range(testIter):
+        
+        a_np = np.random.randn(m, k)
+        b_np = np.random.randn(k, n)            
+
+        a_mat = _matrix.Matrix(m, k, a_np.flatten(order='C')) # row major, C for C-style
+        b_mat = _matrix.Matrix(k, n, b_np.flatten(order='C'))
+        
+        start = time.time()
+        c_mat_naive = _matrix.multiply_naive(a_mat, b_mat)
+        end = time.time()
+        elapsed_naive += end - start
+
+        a_np = np.random.randn(m, k)
+        b_np = np.random.randn(k, n)            
```

Comment:
@yanagiragi  You use two different data for testing two functions.

The goal is to see the performance difference, we would like to decrease any factors that affect the result.
Therefore, you would like to create data sets first, and use the same data sets for testing.

Reply:
you may need to fix this to get 5.

Reply:
You were right. Will fix!